changelog.d/14400.misc

```diff
@@ -0,0 +1 @@
+Remove the `worker_main_http_uri` configuration setting. This is now handled via internal replication.
```

Comment:
Typo in issue title and description:

`worker_http_main_uri` -> `worker_main_http_uri`

Reply:
That's a whoops. Thanks for catching that! Fixed up.